README.md

```diff
@@ -231,30 +255,6 @@ module.exports = {
 }
 ```
 
-#### External Dependencies
```

Comment:
why was this section removed?

Reply:
because externalDependencies option was removed ?

Reply:
Ah, right you are :smile: